galley/pkg/config/source/kube/apiserver/source.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
```

Comment:
do we need the `apiserver` subpackage? I would think that this type should be called `kube.Source`, no? I think WRT a source `kube` and `apiserver` mean the same thing.

Reply:
The kube/ package has other sources as sub packages. There is an in-memory based one, and file based one under this folder, because all of these use K8s data model (i.e. yaml file). I'd prefer to keep all source implementations this way to keep things more uniform.

Having said that, we can change the name apiserver to be something else, if you have a better suggestion.

Reply:
ok makes sense ... I think it's fine.